tests/snappi_tests/utils/metrics.py

```diff
@@ -0,0 +1,116 @@
+"""
+This file defines the classes receiving metrics from snappi tests and processing them.
+"""
+import logging
+import json
+import datetime
+import time
+
+from copy import deepcopy
+from pprint import pprint
+from typing import Dict, Final, List, Union
+
+# Only certain labels are allowed
+METRIC_LABEL_TEST_TESTBED: Final[str] = "test.testbed"
```

Comment:
METRIC_LABEL_TESTBED

Reply:
As we discussed, the first few labels applied to all metrics of the test, so we named them METRIC_LABEL_TEST_**:
METRIC_LABEL_TEST_TESTBED: Final[str] = "test.testbed"
METRIC_LABEL_TEST_BUILD: Final[str] = "test.os.version"
METRIC_LABEL_TEST_CASE: Final[str] = "test.testcase"
METRIC_LABEL_TEST_FILE: Final[str] = "test.test_file"
METRIC_LABEL_TEST_JOBID: Final[str] = "test.job_id"

The next few labels are used to identify a device and a component in a device, so we named them METRIC_LABEL_DEVICE_**:
METRIC_LABEL_DEVICE_ID: Final[str] = "device.id"
METRIC_LABEL_DEVICE_PORT_ID: Final[str] = "device.port.id"
METRIC_LABEL_DEVICE_PSU_ID: Final[str] = "device.psu.id"
METRIC_LABEL_DEVICE_QUEUE_ID: Final[str] = "device.queue.id"
METRIC_LABEL_DEVICE_SENSOR_ID: Final[str] = "device.sensor.id"

Is METRIC_LABEL_TEST_BED a better name?

Reply:
since the testbed is used as a term, so METRIC_LABEL_TESTBED should be good enough.